bindings/2.200/GeometryDash.bro

```diff
@@ -1037,7 +1037,7 @@ class GJBaseGameLayer : cocos2d::CCLayer, TriggerEffectDelegate {
 	// TodoReturn asyncGLoaded(int);
 	void createPlayer() = win 0x18fed0;
 	// TodoReturn flipFinished();
-	// TodoReturn handleButton(bool, int, bool) = win 0x1b2880;
+	void handleButton(bool push, PlayerButton button, bool player1) = win 0x1b2880;
```

Comment:
this will very likely cause a link error on android, since the symbol has it as an int, not PlayerButton (PlayerButton is an enum class, so its a different type).
maybe add it as a overload or something, though you wouldnt be able to use it for hooking ofc

Reply:
just tried, can confirm it does indeed cause a linker error if you try to use it